### ISSUE TITLE
yeah idk, just baaaad (everything seems broken)

### ISSUE THREAD

**Issue by AeSix** [TheBusyBiscuit/ExoticGarden#44]
Issue:
plants don't grow; manually planted or naturally generated.
foods aren't edible; named properly, but only act as a skull.

Explained:
When I place any plant and wait, the plant grows into an oak.  When I traverse to newly generated areas, no EG plants exist.  When attempting to consume foods, there is no action, no error in console. There is an error in the console for plants, however.  It is below. 

Errata:
I have included a list of all of the plugins on the server.  WorldEdit was added in an attempt to resolve the issue, knowing there is a dependency for it in one of the plugins' pom.xml.  I attempted to build EG against Spigot 1.13.2, but failed (I'm not a java dev... yet) 

I pulled copies of the plugins from https://thebusybiscuit.github.io/builds/
exceptions are for ZeldaHearts, which I've had from much earlier versions of MC; and for Barrels, which attempted to self-update, failed, and I downloaded from [the url provided in console.](https://media.forgecdn.net/files/2639/58/Barrels_v1.13.jar) [CS-CoreLib - Updater] Downloading Barrels v1.13

Versions:
 git-Spigot-f56e2e7-7fc66b2 (MC: 1.13.2) (Fresh build right before making this to verify, older versions affected as well)
 WorldEdit version 7.0.0-SNAPSHOT;3997-ba8bde0  (*with and without worldedit installed*)
 CS-CoreLib version DEV - 33 (git e509a156)
 Slimefun version DEV - 65 (git 81dc3e4a) 
 ExoticGarden version DEV - 3 (git 8e3785ee)

additional plugins; listed for informational purposes:
 SlimefunLuckyBlocks version 1.1 (luckblocks-sf-2.jar)
 ChestTerminal version 1.0 (ChestTerminal-3.jar)
 Barrels version 1.12 (Barrels_v1.13.jar)
 ZeldaHearts version 2.0.2 (**also does not work**)
 ModReq version 1.1 (author: Aladram)
 PermissionsEx version 1.23.4
 Vault version 1.6.1
 MultiHome version 0.5 (author MadManMarkAu)
 ProtocolLib version 4.4.0-SNAPSHOT-b398

Console output:
```
[23:01:58 ERROR]: Could not pass event ChunkPopulateEvent to ExoticGarden vDEV - 3 (git 8e3785ee)
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:309) ~[spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) ~[spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) ~[spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at net.minecraft.server.v1_13_R2.Chunk.addEntities(Chunk.java:838) ~[spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_191]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:1556) ~[spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at net.minecraft.server.v1_13_R2.MinecraftServer.postToMainThread(MinecraftServer.java:1565) ~[spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at net.minecraft.server.v1_13_R2.ChunkProviderServer.a(ChunkProviderServer.java:204) ~[spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:602) [?:1.8.0_191]
        at java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:614) [?:1.8.0_191]
        at java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:1983) [?:1.8.0_191]
        at net.minecraft.server.v1_13_R2.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:112) [spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at net.minecraft.server.v1_13_R2.PlayerChunk.a(SourceFile:87) [spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at net.minecraft.server.v1_13_R2.PlayerChunkMap.flush(PlayerChunkMap.java:135) [spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at net.minecraft.server.v1_13_R2.WorldServer.doTick(WorldServer.java:291) [spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:952) [spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
Caused by: java.lang.ClassCastException: org.bukkit.craftbukkit.v1_13_R2.block.CraftBlockState cannot be cast to org.bukkit.block.Skull
        at me.mrCookieSlime.ExoticGarden.PlantsListener.onGenerate(PlantsListener.java:145) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor16.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_191]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_191]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:305) ~[spigot-server.jar:git-Spigot-f56e2e7-7fc66b2]
        ... 20 more
```

**Comment by TheBusyBiscuit** [comment]
ExoticGarden is for MC 1.12, not 1.13, it has not been updated.

**Comment by AeSix** [comment]
Thank You,
I mis-understood as it is available with other recently updated-to-1.13 plugins.
